_docs/runtime/installation.md

```diff
@@ -1,10 +1,91 @@
 ---
-title: "Installation and configuration"
+title: "Runtime installation and configuration"
 description: ""
 group: runtime
 toc: true
 ---
 
-Coming soon
 
-Documentation on how to install it, what are the prerequisites for installing it and what are the different options for various Cloud providers
+Codefresh runtime installs Codefresh Software Development Platoform (CSDP), comprising Argo CD ecosystem components and Codefresh-specific components. The Argo CD components are derieved from a conformed fork.
+
+There are two parts to installing runtimes:
+1. Installing the Codefresh CLI, a one-time action, typically required only for initial CDSP setup.
+2. Installing the Codefresh runtime from the CLI. You can install and provision as many runtimes as you need to. Every runtime is installed at the cluster-level.
+
+Verify [system requirements]({{site.baseurl}}/docs/runtime/monitor-manage-runtimes)
+
+
+### Where do you install runtimes?
+* If this is the first runtime you are installing, on the Welcome page, select **+ Install Runtime**.
+* To install additional runtimes, select **Account Settings**, and then from the sidebar, select **Configuration > Runtimes**. Then from the top-right, select **+ Add Runtime**.
+
+### Installing the Codefresh CLI
+* Codefresh CLI mode  
+  Install the Codefresh CLI using the option that best suits you: Curl, Brew, or standard download. If you are not sure which OS to select for Curl and Brew, simply select one, and Codefresh automatically identifies and selects the right OS for CLI installation.
+
+
+  
+### Installing the Codefresh runtime
+
+#### Runtime components
+ CSDP  create will create a repo (if not exist) and bootstrap (if needed) a new installation, levering the argocd-autopilot. The CLI will determine whether the new runtime is an overlay of an existing runtime or a completely new runtime.
+
+ 
+**Argo CD components**
+
+* Two Github repositories, one for the runtime’s infrastructure, and one for the first GitOps source defined through the default Git Source entity
+* Project, comprising an Argo CD AppProject and an ApplicationSet
+* Installations of the following applications in the project:
+  * Argo CD 
+  * Argo Workflows 
+  * Argo Events
+  * Argo Rollouts
+  
+**Codefresh-specific components**
+* Codefresh Applications in the Argo CD AppProject:  
+  * ArgoCD Agent that feeds GitOps data into CSDP
+  * App-proxy facilitating behind-firewall access to Git 
+  * Git Source entity that references a specific directory on a different repository
+*  A Service that pulls the changes from the repository defined in the Git Source, analyses them, and generates events for the created/updated/deleted files to CSDP
```

Comment:
remove
`A Service that pulls the changes from the repository defined in the Git Source, analyses them, and generates events for the created/updated/deleted files to CSDP`

we don't have this

Reply:
Done